### ISSUE TITLE
AllowJs compiler option causes a crash with Error: ENOENT: no such file or directory, open 'fs' 

### ISSUE THREAD

**Issue by guncha** [s-panferov/awesome-typescript-loader#147]
I'm seeing the latest awesome-typescript-loader crash when the allowJs compiler option is turned on and when trying to import a Node module like `fs`.

Input file:

```typescript
/// <reference path="./typings/index.d.ts"/>

import * as fs from "fs"

console.log("hello, it's a me")
console.log(fs.readFileSync("main.ts"))
```

tsconfig.json:

```json
{
    "compilerOptions": {
        "allowJs": true
    },
    "exclude": [
        "node_modules",
        "typings/browser",
        "typings/browser.d.ts"
    ]
  }
```

webpack.config.js:

```javascript
var webpack = require("webpack");
var path = require("path");

module.exports = {
	entry: "./main.ts",
	output: {
		filename: "bundle.js"
	},
	module: {
		loaders: [
			{
        test: /\.ts$/,
        loader: "awesome-typescript-loader"
      }
		]
  }
}
```

Crashes with:
```
Error: ENOENT: no such file or directory, open 'fs' Error: ENOENT: no such file or directory, open 'fs'
    at Error (native)
Hash: 396f0bfb9d565b6f60f0
Version: webpack 1.13.1
Time: 1217ms
    + 1 hidden modules

ERROR in ./main.ts
Module build failed: Error: ENOENT: no such file or directory, open 'fs'
    at Error (native)
```

Here's a full repro: https://github.com/guncha/awesome-typescript-issue

**Comment by s-panferov** [comment]
@guncha sorry for the delay, thanks for the issue. I'll try to take a look soon.

**Comment by s-panferov** [comment]
I think it should be fixed in `awesome-typescript-loader@0.19.1`, please try.

**Comment by guncha** [comment]
Awesome, works like a charm!! I appreciate all you work.
